libvips/foreign/heifsave.c

```diff
@@ -651,6 +668,12 @@ vips_foreign_save_heif_class_init( VipsForeignSaveHeifClass *class )
 		G_STRUCT_OFFSET( VipsForeignSaveHeif, speed ),
 		0, 9, 5 );
 
+	VIPS_ARG_STRING( class, "encoder", 18,
+		_( "encoder" ),
+		_( "select encoder to use" ),
```

Comment:
nit: argument labels and descriptions should start with a leading capital letter. 
```suggestion
		_( "Encoder" ),
		_( "Select encoder to use" ),
```